examples/miscellaneous-examples.md

```diff
@@ -183,7 +183,7 @@ This quick & dirty swear detector takes an array of swear words we don't want to
 
 ```javascript
 const swearWords = ["darn", "shucks", "frak", "shite"]; // Make sure all of the words are lowercased only.
-if( swearWords.some(word => message.content.toLowerCase().includes(word.toLowerCase())) ) { // Lowercase the message content for better matching
+if (swearWords.forEach(word => message.content.toLowerCase().includes(word.toLowerCase()))) { // Lowercase the message content for better matching
```

Comment:
Not sure why this was changed from `.some()` to `.forEach()`, as the purpose of this snippet is to delete the message _if any of those words are found_, not to loop through all of them.